pkg/server/login/locales.go

```diff
@@ -37,7 +37,12 @@ var supportedLangs = []language.Tag{
 }
 
 var locale_en = map[string]string{
-	"": "",
+	"LogInToYourAccount": "Log in to your account",
```

Comment:
could you please put these to 4 spaces, so it align with go's standard

Reply:
ok I ran `gofmt` on that file. Let me know if it's still not correct.

Reply:
looks fine 👍